src/shapes/Polyline.ts

```diff
@@ -86,8 +101,9 @@ export class Polyline extends FabricObject {
    *   top: 100
    * });
    */
-  constructor(points: XY[] = [], { left, top, ...options }: any = {}) {
+  constructor(points: XY[] = [], options: Props = {} as Props) {
     super({ points, ...options });
+    const { left, top } = options;
```

Comment:
This change is unwanted IMO
Why should we set position before bbox if we do it durring/after bbox calc?

Reply:
because types.